src/pages/ServiceDetails/ServiceInfo/ServiceInfoDestinationPolicies.tsx

```diff
@@ -64,37 +34,45 @@ class ServiceInfoDestinationPolicies extends React.Component<ServiceInfoDestinat
+                      <strong>simpleCb</strong>
+                      <ul style={{ listStyleType: 'none' }}>
+                        {dPolicy.circuitBreaker.simpleCb.maxConnections ? (
+                          <li>[maxConnections] {dPolicy.circuitBreaker.simpleCb.maxConnections}</li>
+                        ) : null}
+                        {dPolicy.circuitBreaker.simpleCb.httpMaxPendingRequests ? (
+                          <li>[httpMaxPendingRequests] {dPolicy.circuitBreaker.simpleCb.httpMaxPendingRequests}</li>
+                        ) : null}
+                        {dPolicy.circuitBreaker.simpleCb.httpMaxRequests ? (
+                          <li>[httpMaxRequests] {dPolicy.circuitBreaker.simpleCb.httpMaxRequests}</li>
+                        ) : null}
+                        {dPolicy.circuitBreaker.simpleCb.sleepWindow ? (
+                          <li>[sleepWindow] {dPolicy.circuitBreaker.simpleCb.sleepWindow}</li>
+                        ) : null}
+                        {dPolicy.circuitBreaker.simpleCb.httpConsecutiveErrors ? (
+                          <li>[httpConsecutiveErrors] {dPolicy.circuitBreaker.simpleCb.httpConsecutiveErrors}</li>
+                        ) : null}
+                        {dPolicy.circuitBreaker.simpleCb.httpDetectionInterval ? (
+                          <li>[httpDetectionInterval] {dPolicy.circuitBreaker.simpleCb.httpDetectionInterval}</li>
+                        ) : null}
+                        {dPolicy.circuitBreaker.simpleCb.httpMaxRequestsPerConnection ? (
+                          <li>
+                            [httpMaxRequestsPerConnection]{' '}
+                            {dPolicy.circuitBreaker.simpleCb.httpMaxRequestsPerConnection}
+                          </li>
+                        ) : null}
+                        {dPolicy.circuitBreaker.simpleCb.httpMaxEjectionPercent ? (
+                          <li>[httpMaxEjectionPercent] {dPolicy.circuitBreaker.simpleCb.httpMaxEjectionPercent}</li>
+                        ) : null}
+                        {dPolicy.circuitBreaker.simpleCb.httpMaxRetries ? (
+                          <li>[httpMaxRetries] {dPolicy.circuitBreaker.simpleCb.httpMaxRetries}</li>
+                        ) : null}
+                      </ul>
                     </li>
-                  )}
-                  {!dPolicy.circuitBreaker.simpleCb.httpMaxEjectionPercent ? null : (
-                    <li>httpMaxEjectionPercent: {dPolicy.circuitBreaker.simpleCb.httpMaxEjectionPercent}</li>
-                  )}
-                  {!dPolicy.circuitBreaker.simpleCb.httpMaxRetries ? null : (
-                    <li>httpMaxRetries: {dPolicy.circuitBreaker.simpleCb.httpMaxRetries}</li>
-                  )}
+                  ) : null}
+                  {dPolicy.circuitBreaker.custom ? <li>[custom] {dPolicy.circuitBreaker.custom}</li> : null}
```

Comment:
Is it either "simpleCb" or "custom"?
Looking at the information displayed, I wonder if we could put a generic introspection algorithm instead of this, to show circuit breaker info.

Reply:
(except if there are some information that we do _not want_ to display)

Reply:
In practice I think only one, looking at documentation, it can be both.
So, I decided to show them if present, at least in this iteration.
A generic component was also my first thought, but then I finished that we might want fine granularity about what to show, what not, and I decided to cover all documented fields.